cip-CC-FEE-REMOVAL/cip-CC-FEE-REMOVAL.md

```diff
@@ -0,0 +1,276 @@
+<pre>
+  CIP:  CIP XXXX
+  Title: Canton Coin Fee Removal
+  Author:
+    Simon Meier
+  License: CC0-1.0
+  Status: Draft
+  Type: Tokenomics
+  Created: 2025-08-19
+  Approved:
+</pre>
+
+## Abstract
+
+This CIP proposes to remove Canton Coin fees and adjust holding fees so that
+users do not have to pay fees when transferring Canton Coin and application
+developers can build applications without special code to deal with these fees.
+This CIP further proposes to have users collect their validator rewards directly
```

Comment:
remove from abstract

Reply:
formulate it as technical cleanup

Reply:
state this as a positive assertion that users that purchase traffic will get credit for the activity. This is a related change that's now possible because we are removing usage fees.